.github/workflows/main.yml

```diff
@@ -22,8 +22,8 @@ on:
       - main
       - 'stable/**'
   schedule:
-    # run every day at 1AM
-    - cron: '0 1 * * *'
+    # run every day at 2AM
+    - cron: '0 2 * * *'
```

Comment:
Afaik, currently Manoel is still being informed about scheduled CI runs failing at night.
Changing this timestamp here should change the notification settings such that I am getting notified (I previously read about this in the Github docs somewhere, but cannot find it now; but basically there is no way to manually subscribe to notifications triggered by scheduled CI runs...)